src/blocks/slider/editor.scss

```diff
@@ -59,31 +59,39 @@
 			top: 2px;
 			right: 2px;
 			background:#fff;
-			border: 1px solid rgba(66,88,99,.4);
-			border-radius: 4px;
+			border: 1px solid rgba(30, 30, 30, 1);
+			border-radius: 2px;
 			transition: box-shadow .2s ease-out;
 			margin: 8px;
 			display: inline-flex;
 			z-index: 20;
 
 			.components-button {
-				color: rgba(14,28,46,.62);
-				padding: 2px;
-				height: 24px;
+				color: rgba(30, 30, 30, 1);
+				padding: 6px;
+				min-width: 36px;
 
 				&:hover {
 					box-shadow: none;
 				}
 			}
 		}
 
-		.block-editor-rich-text {
+		.wp-block-themeisle-editor-rich-text {
```

Comment:
We don't need to use custom class. We can change it to `.block-editor-rich-text__editable`

Now I remember how this became an issue. Block editor recently changed a lot of classes.